photon-api/src/main/scala/com/linkedin/photon/ml/optimization/DistributedOptimizationProblem.scala

```diff
@@ -81,7 +81,7 @@ protected[ml] class DistributedOptimizationProblem[Objective <: DistributedObjec
    * @param coefficients The feature coefficients means
    * @return An optional feature coefficient variances vector
    */
-  override def computeVariances(input: RDD[LabeledPoint], coefficients: Vector[Double]): Option[Vector[Double]] = {
+  override def computeVariances(input: RDD[LabeledPoint], coefficients: BVector[Double]): Option[BVector[Double]] = {
```

Comment:
Why do we need this?

Reply:
It is not necessary in this fix. I copy it from the migration codes and we would need spark ml vector so we need to rename breeze vector to BVector. We could remove it in this commit.

Reply:
I think we can remove it to keep this commit clean.